src/main/java/nextstep/lms/domain/Student.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.lms.domain;
+
+import nextstep.lms.enums.StudentStatusEnum;
+
+import java.util.Objects;
+
+public class Student {
+    private final Long userId;
+    private final Long sessionId;
+    private StudentStatusEnum studentStatusEnum;
+
+    public Student(Long userId, Long sessionId) {
+        this(userId, sessionId, StudentStatusEnum.APPLIED.name());
+    }
+
+    public Student(Long userId, Long sessionId, String studentStatus) {
+        this.userId = userId;
+        this.sessionId = sessionId;
+        this.studentStatusEnum = StudentStatusEnum.valueOf(studentStatus);
+    }
+
+    public boolean isSelected() {
+        return studentStatusEnum == StudentStatusEnum.SELECTED;
+    }
+
+    public Student setStatus(StudentStatusEnum studentStatus) {
+        this.studentStatusEnum = studentStatus;
+        return this;
+    }
```

Comment:
setter를 제공하기 보다 특정 상태를 바꾸는 메서드를 제공하면 어떨까요?

```suggestion
    public Student select() {
        this.studentStatusEnum = StudentStatusEnum.SELECTED;
        return this;
    }
```